src/controller/prodController.js

```diff
@@ -0,0 +1,78 @@
+import jsend from 'jsend';
+import db from '../database/models/index.js';
+
+async function registration(req, res) {
+  // A seller should be able to Create/Add a product
+
+  try {
+    // receive body & Validate user input
+    const {
+      Name, description, categoryId, pictureUrls, expiryDate, price, Instock, available,
+    } = req.body;
+    if (
+      !Name
+      || !description
+      || !categoryId
+      || !price
+      || !pictureUrls
+      || !expiryDate
+      || !Instock
+      || !available
+    ) {
+      return res.status(400).json(jsend.fail({ message: 'Invalid Input😥' }));
+    }
+    // verify the product is not sold by the seller
+    const productExists = await db.Product.findOne({
+      where: { vendor_id: req.user.id, Name },
+    });
+    if (productExists) {
+      return res.status(409).json(
+        jsend.fail({
+          message: 'Seller already sells this product😥',
```

Comment:
I think the message should be "Seller already have this product😥"

Reply:
Done!